### ISSUE TITLE
Lack of full screen option.

### ISSUE THREAD

**Issue by JBWKZsf** [rxhanson/Rectangle#739]
I only see the option to maximize the screen. I think a short cut to make a window full screen will be very helpful

**Comment by rxhanson** [comment]
It’s not in rectangle because it’s already in macOS. Cmd + ctrl + F, and I think on newer installations of macOS it is just fn+f

**Comment by rxhanson** [comment]
Here's how to do it in macOS:

1. Open System Preferences > Keyboard > Keyboard Shortcuts
2. Go down to Application Shortcuts on the left, click the + sign.
3. In the drop-down box, make sure "All Applications" is selected, then add `Enter Full Screen` and assign your shortcut to that.
4. Repeat the above and add `Exit Full Screen` and again assign your same shortcut to that.

Note that this is likely localized so you might have to account for that if English is not your system language.

**Comment by rluvaton** [comment]
Thank you for the effort of writing this, I did that and I remembered it didn't work for all apps, checked it again now, and looks like it works, so thanks!

**Comment by rluvaton** [comment]
Apparently, your proposed solution is not working in every application, for example - VS Code doesn't work, this seems to for some apps only, so it would be wonderful if we could re-open this issue

**Comment by rxhanson** [comment]
For VS Code, just follow the same steps but add `Full Screen` as the menu item.

I realize it would be easier and more portable if this were a part of Rectangle, but I'm not convinced that I want to add it in at this time.

**Comment by rluvaton** [comment]
> I realize it would be easier and more portable if this were a part of Rectangle, but I'm not convinced that I want to add it in at this time.

Is it time related? If so I can try and create a PR

**Comment by rxhanson** [comment]
The main limiting factor in my opinion is the UI. The most common request for Rectangle is always to add more window actions. Each individual action like this one makes sense, but they add up. Going down this path ultimately leads to the list view that I implemented for Rectangle Pro. Initially I figured I would go the same route for Rectangle, but my perspective is that Rectangle is liked because it is simple. If I were to put Rectangle Pro on my mom’s computer, I think the list view would not be as appealing as the current UI for her. The current UI also feels like the other apps that came before it. My intention is not to keep Rectangle frozen, but to only add things to the UI that are seriously desired by a lot of users. 

Provided that there exists macOS behavior for full screen, it makes it a little harder for me to justify adding it in.  

I have no issues adding it to Rectangle Pro, since it already has the list view and it caters to a different audience.
